helm/synthetic-image-publication-monitor/templates/deployment.yaml

```diff
@@ -41,7 +41,7 @@ spec:
               name: global-config 
               key: aws.s3.image.binaries.bucket
         - name: TEST_UUID
-          value: "6a70679a-fd9b-4d64-9bec-2d340fb75021"
+          value: "{{ .Values.env.imageUuid }}"
```

Comment:
I'm thinking it would be better to add this to the global config, instead of a helm env. To keep only there the per env config.